pkg/cmd/cli/cmd/process.go

```
@@ -112,128 +111,129 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 	}
 
 	var (
-		templateObj *api.Template
-		mapping     *meta.RESTMapping
+		objects []runtime.Object
+		mapping *meta.RESTMapping
 	)
 
+	version, kind, err := mapper.VersionAndKindForResource("template")
+	if mapping, err = mapper.RESTMapping(kind, version); err != nil {
+		return err
+	}
+
+	outputVersion := kcmdutil.OutputVersion(cmd, mapping.APIVersion)
+	raw := kcmdutil.GetFlagBool(cmd, "raw")
+
+	infos, err := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
```

You are ignoring this error.